.editorconfig

```
@@ -0,0 +1,3 @@
+[*.json]
```

Mark this directory as the project root to prevent editorconfig from traversing parents.
```suggestion
root = true

[*.json]
```